lucene/CHANGES.txt

```diff
@@ -45,6 +45,11 @@ Improvements
   docs on equal scores. Also, remove the ability of TopDocs.merge to set shard
   indices (Atri Sharma, Adrien Grand, Simon Willnauer)
 
+* LUCENE-8939: Introduce Shared Count Based Early Termination Across Multiple Slices
```

Comment:
We don't capitalize all words in our `CHANGES.txt`, just the first one and any proper nouns, etc. (like a normal sentence).

Reply:
Fixed, thanks